src/kademlia/node.cpp

```diff
@@ -1152,4 +1157,14 @@ void node::incoming_request(msg const& m, entry& e)
 		return;
 	}
 }
+
+void node::update_counters()
+{
+	const dht_storage_counters& dht_cnt = m_storage->counters();
+	m_counters.set_value(counters::dht_torrents, dht_cnt.torrents);
```

Comment:
i.e. is this the only place m_counters is used? (if so, it would be nicer to pass it in as an argument I think)